jdi-light-html-tests/src/test/java/io/github/epam/example/common/ButtonExampleTests.java

```diff
@@ -0,0 +1,61 @@
+package io.github.epam.example.common;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.elements.common.Alerts.validateAlert;
+import static io.github.com.StaticSite.html5Page;
+import static io.github.com.pages.HtmlElementsPage.dblClickButton;
+import static io.github.com.pages.HtmlElementsPage.redButton;
+import static io.github.epam.html.tests.site.steps.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Test examples for Button with JDI Light asserts
+ *
+ */
+public class ButtonExampleTests implements TestsInit {
+
+    String text = "Big Red Button-Input";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4713869)